src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -89,4 +90,19 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Parses {@code String category} into a {@code Category}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code category} is invalid.
+     */
+    public static Category parseCategory(String category) throws ParseException {
+        requireNonNull(category);
+        String trimmedCat = category.trim();
+        if (!Tag.isValidTag(trimmedCat)) {
```

Comment:
Is this supposed to check for isValidTag? Also, good to maintain consistency in naming conventions (ie, using the full name of attributes such as address and category instead of addr or cat, can sometimes be confusing)

Reply:
o yeah i copy paste wrongly later i do